unlock-app/src/__tests__/utils/types.test.js

```diff
@@ -0,0 +1,20 @@
+import { transactionTypeMapping } from '../../utils/types'
+import { TransactionType } from '../../unlockTypes'
+
+describe('transactionTypeMapping', () => {
+  it('should map the types from unlock-js', () => {
+    expect.assertions(4)
+    expect(transactionTypeMapping('LOCK_CREATION')).toEqual(
```

Comment:
mapping from raw strings makes me feel nervous, but as long as this is the only place they happen, and we always use this file, it should be fabulous :)

Reply:
Yeah, I should have added a comment that we should actually load the values from the expored types in unlock-js and then test on all of them.